src/main/java/org/tron/common/overlay/discover/node/NodeHandler.java

```diff
@@ -232,6 +232,9 @@ public void handleNeighbours(NeighborsMessage msg) {
   }
 
   public void handleFindNode(FindNodeMessage msg) {
+    if(msg.getFrom().getHost().equals("127.0.0.1")){
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this IP "127.0.0.1" address configurable. ([squid:S1313](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1313))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1164&issues=AWTLs5Iy6dA5b-W28RLt&open=AWTLs5Iy6dA5b-W28RLt)

[ISSUE_KEY]: AWTLs5Iy6dA5b-W28RLt